components/data/data-services/org.wso2.micro.integrator.dataservices.core/src/main/java/org/wso2/micro/integrator/dataservices/core/DBDeployer.java

```diff
@@ -841,6 +841,11 @@ private AxisService createDBService(String configFilePath,
 			/* save the data service object in the AxisService */
 			axisService.addParameter(DBConstants.DATA_SERVICE_OBJECT, dataService);
 
+			String swaggerResourcePath = dataService.getSwaggerResourcePath();
+			if (swaggerResourcePath != null && !swaggerResourcePath.isEmpty()) {
```

Comment:
Can we use org.apache.commons.lang.StringUtils.isNotEmpty(swaggerResourcePath) for this?

Reply:
+1 fixed